business-wiki-frontend/shared/components/Footer.tsx

```diff
@@ -0,0 +1,12 @@
+import moduleStyles from "../styles/Footer.module.scss";
+
+const Footer = () => {
+  return (
+    <footer className={moduleStyles.footer}>
```

Comment:
Destrukturyzacja.

Reply:
Resolved in e693bf16ffbe06814090d742bedcd736a538310d